components/centraldashboard/public/components/resources/kubeflow-icons.js

```diff
@@ -16,14 +29,23 @@ import bolt from '../../assets/icons/ic_bolt.svg';
 function stripSVG(svg) {
     const content = svg
         .replace(/\r?\n|(?=>)\s+(?=<)/g, ' ')
-        .replace(/.*?<svg.*?>(.+?)<\/svg>/, '$1');
+        .replace(/^.*?<svg.*?>(.+?)<\/svg>.*$/, '$1');
     if (content == svg) throw Error('Invalid SVG data');
```

Comment:
How does this behave if someone mistakenly puts a bad file in the icons folder? If it results in a page that doesn't render  on development and fails unit tests, that would be perfect. The main concern is just that we don't end up with someone being able to mistakenly put something there that gets past tests and would result in a faulty image.

Reply:
Now comes with a verbose error in console. Should be enough to help the dev debug the issue, but not break the page itself.